src/workerd/server/tests/python/random/worker.py

```diff
@@ -0,0 +1,41 @@
+from random import random, randbytes, choice
+
+try:
+  random()
+except RuntimeError:
+  pass
+else:
+  assert False
+
+try:
+  randbytes(5)
+except RuntimeError:
+  pass
+else:
+  print("randbytes() no")
```

Comment:
Probably not worth restarting CI for this, but Is this print() supposed to be here?